tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceDialogTest.java

```diff
@@ -426,6 +429,19 @@ public void testActivateWholeWordsAndSearchForTwoWords() {
 		assertFalse(dialog.wholeWordCheckBox.getEnabled());
 	}
 
+	@Test
+	public void testActivateDialogWithSelectionActive() {
+		openTextViewer("text" + System.lineSeparator() + "text" + System.lineSeparator() + "text");
+		fTextViewer.setSelectedRange(5, 9);
+		openFindReplaceDialogForTextViewer(false);
+
+		assertFalse(dialog.globalRadioButton.getSelection());
+		dialog.findCombo.setText("text");
+		dialog.replaceAllButton.notifyListeners(SWT.Selection, null);
```

Comment:
Please use `select(dialog.replaceAllButton())`

Reply:
Addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/d2b76269413f3578e8a605d2410049c3801569be